yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1529,6 +1529,11 @@ To manually modify the schema, please refer: <a class="highlight-link" href="htt
 		Type: SizingNotes,
 		Text: `Reference and System Partitioned tables are created as normal tables, but are not considered for target cluster sizing recommendations.`,
 	}
+
+	REDUNDANT_INDEX_ESTIMATED_TIME = NoteInfo{
+		Type: SizingNotes,
+		Text: `Estimated time taken for data import (Excluding redundant indexes) is calculated without considering redundant indexes as those indexes are removed during export schema phase automatically.`,
```

Comment:
In the assessment report, we will continue reporting both times - with and without redundant indexes, right?
This seems a bit repeatitive.

Some suggestions:
```
Times excluding redundant indexes reflect actual imports, as those indexes are automatically removed during schema export.

OR

Redundant indexes are dropped in the schema export phase, so the ‘without’ estimate is the effective import time.
```